ufm-test-services/metamask/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:18
+
+WORKDIR /app
+
+COPY ./ ./
```

Comment:
is this meant to copy the entire optimism repo or is it meant to only be copying ufm-test-services/metamask/dockerfile?

If it's only meant to be run in context of only this service you should consider removing ufm-test-services/metamask from the pnpm-workspace and giving it it's own pnpm-lock.yaml

Reply:
I see from https://github.com/ethereum-optimism/optimism/pull/7152 that this is a standalone service in the dockerfile. You are not copying the pnpm-lock.yaml. For this reason it would be a good idea to make this package have it's own pnpm-lock.yaml and remove from pnpm workspaces

Reply:
Update `metamask` dir to generate it's own `pnpm-lock.yaml`